frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -2005,6 +2065,68 @@ mod tests {
 		})
 	}
 
+	#[test]
+	fn challenge_solution_works() {
+		ExtBuilder::default().build_and_execute(|| {
+			crate::mock::SignedMaxSubmissions::set(50);
+			roll_to(14);
+			assert_eq!(MultiPhase::current_phase(), Phase::Off);
+
+			roll_to(15);
+			assert_eq!(multi_phase_events(), vec![Event::SignedPhaseStarted { round: 1 }]);
+			assert_eq!(MultiPhase::current_phase(), Phase::Signed);
+
+			let mut solution = raw_solution();
+
+			solution.score.minimal_stake += 1;
+
+			assert!(mock::Balances::usable_balance(&99) == 100);
+
+			assert_ok!(MultiPhase::submit(crate::mock::Origin::signed(99), Box::new(solution),));
+
```

Comment:
please avoid all of these newlines. A newline in a sequence of code should act more or less signify the end of a paragraph.